Sequences/MRML/vtkMRMLVolumeSequenceStorageNode.cxx

```diff
@@ -164,6 +164,21 @@ int vtkMRMLVolumeSequenceStorageNode::ReadDataInternal(vtkMRMLNode* refNode)
   // Read and copy the data to sequence of volume nodes
 #ifdef NRRD_CHUNK_IO_AVAILABLE
   int numberOfFrames = reader->GetNumberOfImages();
+  vtkImageData* imageData = NULL;
+  vtkNew<vtkImageExtractComponents> extractComponents;
+  if (reader->GetIsCompression())
```

Comment:
Property name can be either IsCompressed or UseCompression (not IsCompression). UseCompression is typically used when we can choose to enable/disable compression (e.g., in file writers), so maybe IsCompressed is the better name.

Reply:
ok, fixed already